ui/browser/views/navbar/btn.jsx

```diff
@@ -12,21 +12,41 @@ specific language governing permissions and limitations under the License.
hm2yKjrBizULkaVMXGJVRXnxTRSzVeasLKBQcTHKLAPBE+Dn/OocGTBUlkaq1h1Qdp5xvsz58/n9cwjY2NYo5WHD2+MWdT9MJYiVQJeuDMgYLRHDX5xQUGSwuLS6Z/2vUbNxBbYJlAgyua/OtzRgprnyRUkbxidfPlFic7i6ANlIdIVXKVVjBPlIbJhjkTIlOtXpPe3Nwyv2B6+vRpe3s7QplNti3vJS0HNAFBUiwrcPXTzFFM5ajoRb29fW98JjzHnn2leK2SUCUIEJ4jYoN5aq3xwMHDc+mwYN7e/ehgcKjC4YrEbnClcOefbN/15MkTP4cJsbD9m8ay8oMpq1MxL9Vh4RMuRKsLj5ihn0D08/XZf019MjY4y5anwKvB7btonPDYML0JpGZm5fT398+q47du3V6evAoYucw8h5GYclj4Et9b9uOPd/0NpuHhYezFyg8cWr8hO25xIjw8AgW8UyEWnu3LUqp0xVv/d6U7evwEIg9Aif/g6maKiIaLil+ceGfGQ1Vx7LhmYtpMwciFSOFtIE44darWr2C6d+8+uJlgSI4hN0U6tQDpDCZsoZ89eyY8fGRkNDfPJsx4y4IYt4wTmhXzISZ28d1796fv+9DjIdvmAmzWHA7JXc2ywKrSGBBFYTfqPzC9ejW+Ob8IAawobxOLI/xZ54MHeDJSLAiKMVRGsxOAOhFCYVt+7377/+t4R+cDJDUwhRARut88wT9N5DhUR8CT38RMV69eV6h0wtbMyYaH4AXZ7ReR68OiicSB4JDc3/C5yF1ZYxe33bn7xl2bdVG8TBHmCdum8qSsqz/jJzC9ePEC5xWYys6/HSFrh7OFhCVJcqXnhwoe9+34Jb/8+uvk/l5q+jbCEo08pEfNcxiJrIHOENF2546f7ObKPjuA+SFE3M43ZC8dSQRPN/in1LUZY2NjQk/t9m8weGjeYZ7DiSavXI1Y0x9gutL6HcJwzA/0zZ+aQExQiByns+hmwwU78h1KjR7j522mYjJXVlX7A0wjIyPx77yLF41e+V/DJhzZVBQj4GeHT/KyhqhgybtJSGb6Q9IyO2eT45jM7/wTVnDhsFbwSV7Yfk/QyJSoUPADmBA2HVSq/RImx87Ay4lHmiArOw8VGT4PU23dGSQI0Cs2T7VwU5Q5cuFPPT0+D9P3//5Bhs38vB9RzzonbOvOnb/g8zDhXAUH2r+vBfN+UD3YUE26c9enPg9TT0+PNSZOjyKeeT+iHmw4sFuT9qHPw4Ti16VJyaji8FSAjO0MTmNQ8BRujsLPXugj3WAVqg6Ro3kyPOzbMCH9+uFHme7c0Al1kjixx+F5yEQBJ2p9DCYLcqcwQzJR5Y2TY4+nhbR6MyxEERJSQTjOw9G1UGHhihI/fBbOp3G04tsw4cABd3eUaoN7RghOSKrQSKQqa0x8WnrGtm3bUVOF6xwX7BfPfHW2ouL41m3bccaHlyuVqzFmHkEKoOOsGh+dnrH+Y1t+WVn5vtLPUH2KDTyOsQGW6Kzrw81gFFctfBsmpDdy8mx4Qa5OHiIsQGkYfticX4idS3t7xzRWtbZ+vy4zC3scLH/u4Un4FLhGhVq3LDmlsrrm9u22qYZ1d/9UVX0S1TWoQBfRMHQTBNfW1vt82e72HX9H2avrjjMRkGGi4zJJ+YHDqFGZoVXj4+O4fgn/5B6SMJxYxWDkl6dq//BwA4d9WJSRWBfRYWNDV1VV4/MwodAdsYsrqh+FewSWKOuhwxV9fY9maxh4yli3Af7J1c4J9Q74FFt+QW9f3wxtw/1MDL+IMT4iRaz4Pg9TySc7AZPoJCGxjiWjsKikq6vbmTSYsAS49Nwe58HnzjXMyrBvm1tUGjHzc28FSIqKSwjTm8tnUdP41dlzzpu3MTsPF7Fd4TjxJzaPuFl68+at2Vr16NHPkVExRrNFLHsCgkIxED4P07aSHYBJrOADyz8ipPWZ2b29vaKYd7q2Dp4JjxW7cDYSQVJWdu7joaG5XbhDmhH5IcL0mg4eOoKLb2IFkmF6IwJnhDsiHvjgsUaTRcQ9Aa4EIpjbu690zlahg3CZCJsI02uqrKp5KzBEnFIvlRbXGkWv4EtIXKoLN4s1bAajBffsKitrnA4PdqBYijC9pn+cqBIRppRVqeKaBx+Ql2dTaURLq+JCX+aGbOcNQ00wbi4QJhfCtDJljegWYpujUIl24IPacORpnbcKqyRiTcLkYzDheymwoIjomXAwQpjmKUz4kkIMG2EiTOJsEZC48jaY8F08hMn3YDpRWe2FMO3bX0aYCJM4MGHxxZehESYfg6mhwY7aNG+DCWVYfwsMJUw+BhO+XTMgSOZtMKFYhTD5HkxNTc2BwYSJMBEmwkSYCBNhIkyEiTARJsJEmAgTYSJMhIkwESbCRJgIE2EiTISJMBEmwkSYCBNhIkyEiTARJsJEmAgTYSJMhIkwEab/tlM3KQ2DURhGt6RNY2KsRcSRFBE3YQWd+4MuwIquQnGouIXiTpJWCLVCAhmaPeSropyHu4B3cLgwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDBBBNMMMEEE0wwwQQTTDDB9EuYHp+e16Jkf3TY/baGu8fj0+ALp9P3KM6CLGwvTgfnl9fdV728vq1HaahVvX56O7n/D5h68ebo4Kj7be/sjU/OVvGZ4mQQZGF7STa8uLoJgqklHmpVfyOb3D38eUxVVeV5MZ9/dL9iNivLMvjCpmlCLWwvL4rF4rP7qrquQ67Ki+Xya8WYJJgEk2CSYBJMgkkwSTAJJsEkwSSYBJNgkmASTIJJgkkwCSbBJMEkmASTBJNgEkyCSYJJMAkmCSbBJJgEkwSTYBJMEkyCSTAJJgkm/XzfL07KBwH/1dEAAAAASUVORK5CYII=";
```

Comment:
Why is this at the bottom of the file?  Also, how about we don't set a background image if it's not passed in instead of using a default (which shouldn't ever be used)?  I was planning to convert the 'pages' button to a Btn for #17 so it could have arbitrary child elements with no icon.  Maybe I should instead make a new component for text buttons but I was thinking we could re-use things like the disabled / active state.